lib/gollum.rb

```diff
@@ -29,9 +29,10 @@ def self.add_filter(pattern, &replacement)
       @@filters[pattern] = replacement
     end
 
-    def self.apply_filters(data)
+    def self.apply_filters(wiki_page, data)
       @@filters.each do |pattern, replacement|
-        data.gsub!(pattern, replacement.call)
+        ps = replacement.parameters.length == 0 ? () : (wiki_page)
```

Comment:
Can you give `ps` a more descriptive name? `params`, say (if understand well)?

Also, the `()` syntax looks a bit unidiomatic. Why not just `nil`, and `wiki_page` (without parentheses)? But maybe I'm missing something about how the splat operator on the next line functions. :)

Reply:
I thought `nil` won't work, sorry for my first PR in Ruby, and thanks for telling me. 😄